tests/v2/test_wakunode.nim

```diff
@@ -789,9 +789,9 @@ procSuite "WakuNode":
       await node1.publish(rlnRelayPubSubTopic, message)
       await sleepAsync(2000.millis)
 
-
+      ## Current circom takes more time to load the circuit..
       check:
-        (await completionFut.withTimeout(10.seconds)) == true
+        (await completionFut.withTimeout(200.seconds)) == true
```

Comment:
Just double checking that `200` is indeed necessary. :) Any idea what the usual successful loading time is, more or less? I know the circom loading takes a long time, but waiting > 3 mins for the test to fail could be frustrating during regular CI.

Reply:
Not necessary anymore with zerokit rln perfomance [fixes](https://github.com/vacp2p/zerokit/pull/26).